internal/cmd/sync_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
The build tags are directives for the compiler, not comments. To indicate this, there should not be a space after `//`.